badge-maker/index.d.ts

```diff
@@ -4,6 +4,10 @@ interface Format {
   labelColor?: string
   color?: string
   style?: 'plastic' | 'flat' | 'flat-square' | 'for-the-badge' | 'social'
+  logo?: string
+  logoPosition?: number
+  logoWidth?: number
```

Comment:
I don't think there is a strong case for exposing `logoPosition` and `logoWidth` in the badge-maker package. `logoPosition` is not exposed at all by shields.io. `logoWidth` kind of is available by accident but it is not documented and we may deprecate or quietly remove it at some point.

Reply:
What do you think about adding but marking `logoWidth` as deprecated? I think it should be more clear.

Also I remove `logoPosition` from `make-badge.js` also as it does not used at all.

Reply:
As things stand right now, `logoWidth` is not exposed in the public API at the library level. If you call `makeBadge()` with a `logoWidth`, it will throw `ValidationError: Unexpected field 'logoWidth'`

I don't think we should expose `logoWidth` as deprecated and then remove it again later. Lets just keep this as it stands (i.e: not exposed) at the library layer.

Reply:
Ok, and what do you think about removing `logoWidth` from `makeBadge` function then as I did in [this commit](https://github.com/badges/shields/pull/9645/commits/c583fc7baa92300d32f3235fd9b9483d3ae51759)?

Reply:
I will reply to this, but I don't have time immediately

Reply:
Although it is not exposed via the NPM package, `logoWidth` is exposed via shields.io.
https://img.shields.io/badge/foo-bar-blue?logo=javascript
https://img.shields.io/badge/foo-bar-blue?logo=javascript&logoWidth=50
I would like to remove it, but just deleting the param here is not in isolation a massively helpful thing to do.

I've raised an issue about this: https://github.com/badges/shields/issues/10285